### DISCUSSION TITLE
Feel free to ask questions.

### DISCUSSION THREAD

**Discussion by Jay13Jeong** [Jay13Jeong/GBFR-logs-anti-cheat#1] [General]
All comments are welcome.

**Comment by ayano2k** [comment]
This is great, I really like being able to determine anomalies.
I would like to be able to add support for attack V+ and battle history illegitimacy.
This is very helpful, thank you very much.
I used a translator. Sorry if I made any mistakes.

**Reply by Jay13Jeong** [reply]
I'm glad it's helping. I'm also considering the features you mentioned. Thank you for the feedback.

**Comment by Nizika569** [comment]
치트상태의 유저를 감지한경우(로비) 진의 경우 어느부분인지 라인이 뜨니까 알 수 있으나
스탯의 경우 게임을 하지 않으면 볼 방법이 없는데, 이를 수정할수 없는지 궁금합니다
